From a66c50c0e8cd3799fc2819835b872ab62419f684 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Tue, 1 Apr 2014 02:53:36 -0400
Subject: [PATCH] policycoreutils: fix TypeError for seobject.py

File "/usr/lib64/python2.7/site-packages/seobject.py", line 109, in log
  message += " sename=" + sename
TypeError: cannot concatenate 'str' and 'NoneType' objects

Uptream-Status: pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 semanage/seobject.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/semanage/seobject.py b/semanage/seobject.py
index f8d2243..66653e6 100644
--- a/semanage/seobject.py
+++ b/semanage/seobject.py
@@ -105,7 +105,7 @@ except:
 
 		def log(self, msg, name = "", sename = "", serole = "", serange = "", oldsename = "", oldserole = "", oldserange = ""):
 			message = " %s name=%s" % (msg, name)
-			if sename != "":
+			if sename != "" and sename != None:
 				message += " sename=" + sename
 			if oldsename != "":
 				message += " oldsename=" + oldsename
-- 
1.7.9.5

